packages/fxa-content-server/app/scripts/lib/email-domain-validator.js

```diff
@@ -50,25 +51,33 @@ const resolveDomain = domain =>
   });
 
 const checkEmailDomain = ($element, view) => {
+  // The two functions below have their tooltip displaying code wrapped in $()
```

Comment:
Gotta love obscure problems like this. Nice debugging. Same exact reason we had to do this here: https://github.com/mozilla/fxa/blob/f98235ed4a226e00df985fb0692947557487a2d8/packages/fxa-content-server/app/scripts/views/base.js#L973